src/lib/multicall/multicall.worker.ts

```diff
@@ -28,7 +28,13 @@ async function run(event) {
       result,
     });
   } catch (error) {
-    postMessage({ id, error: error });
+    postMessage({
+      id,
+      error: {
+        message: error.message,
+        stack: error.stack,
```

Comment:
i think it makes sense to also pas error.name and error.cause

Reply:
Added error.name, but error.cause is often used to pass an original error object which can cause the sendMessage issue again. Let's keep this place simple as possible

Reply:
good point